### PR TITLE
Removes fixed margin between 'Send Link' and 'Enter your password ins…

### LINKED ISSUE #6477: Login: magic link screen is not scrollable
### Expected behavior
On very small screens, the magic link screen should be scrollable as to be able to click the "I'll enter the password" link that should be at the bottom

### Actual behavior
There's no way out, leaving the user with the only option to send the magic link, unless you rotate the screen (which looks and behaves alright).

screen shot 2017-08-03 at 2 34 14 pm

### Steps to reproduce the behavior
1. Use a very tiny screen, for instance a 240x320 emulator
2. install the app
3. try to login entering your wp.com username
4. see the magic link screen appear and you can't do anything else.

### PR DESCRIPTION
**Pull request by baseilos**
Removes fix margin to minimises scrolling space and wraps whole view in ScrollView to allow user scroll on very small screens.

screen shot 2017-09-30 at 16 38 12

screen shot 2017-09-30 at 16 41 40

Fixes #6477 

To test:

### CODE REVIEW

File: WordPress/src/main/res/layout/login_magic_link_request_screen.xml [hunk 1]
```diff
@@ -1,73 +1,79 @@
...
 
-    <FrameLayout
-        android:id="@+id/avatar_container"
-        android:layout_width="@dimen/avatar_sz_login"
-        android:layout_height="@dimen/avatar_sz_login"
-        android:layout_below="@+id/toolbar"
-        android:layout_marginTop="@dimen/margin_extra_extra_large"
-        android:layout_marginBottom="@dimen/margin_extra_large"
-        android:layout_centerHorizontal="true">
+        <include
```
Comment (tonyr59h) [inline WordPress/src/main/res/layout/login_magic_link_request_screen.xml]: The toolbar should be excluded from the `ScrollView` or it ends up scrolling.

File: WordPress/src/main/res/layout/login_magic_link_request_screen.xml [hunk 2]
```diff
@@ -1,73 +1,79 @@
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView
```
Comment (tonyr59h) [inline WordPress/src/main/res/layout/login_magic_link_request_screen.xml]: I don't think the overscroll animation is necessary on this screen. Could you add 
`android:overScrollMode="never"` here?

**Comment by baseilos** [comment]
Thank you for review!

Proposed changes added.

**Comment by mzorz** [comment]
@hypest do you want to take a look into this solution as well? And maybe close PR https://github.com/wordpress-mobile/WordPress-Android/pull/6482 if this solution seems better than what you had in mind 🙇

**Comment by hypest** [comment]
Thanks for the ping @mzorz and of course thanks for contributing @baseilos !

I'd say if @tonyr59h is done with his review we can merge this one! @tonyr59h feel free to do it if so.

I'd recommend we keep your PR open though @mzorz as we might continue work in that. I think that the scrollbar solution of this PR is fine for now but, probably less than ideal for a screen that only tries to display a couple of things.

**Comment by tonyr59h** [comment]
Thanks for improving the app @baseilos! This one is good to go :shipit:

**Pull request merged**